kodi_addon_checker/check_dependencies.py

```diff
@@ -56,7 +56,43 @@
         'krypton': {'min_compatible': '2.1.0', 'advised': '2.25.0'},
         'leia': {'min_compatible': '2.1.0', 'advised': '2.26.0'},
         'matrix': {'min_compatible': '3.0.0', 'advised': '3.0.0'}
-    }
+    },
+    'xbmc.gui': {
+        'gotham': {'min_compatible': '5.0.0', 'advised': '5.0.0'},
```

Comment:
Can you check if it would make sense to default to `AddonVersion` objects here? So that we don't need to parse these for every useage?

Reply:
you can disable checks I suppose. Hence this may not even be used. I think it's a good a idea but out of the scope of this PR